Documentation/cli/README.md

```diff
@@ -32,6 +32,7 @@ Command | Description
 [print](#print) | Evaluate an expression.
 [regs](#regs) | Print contents of CPU registers.
 [restart](#restart) | Restart process from a checkpoint or event.
+[rev](#rev) | Performs the reverse operation of the command specified.
```

Comment:
nit: I would be more clear about this. Something more along the lines of `Reverses the execution of the target program for the command specified.`